### PR TITLE
update lz chainIds and endpoint addresses

### PR DESCRIPTION
**Pull request by pppest**
Add new lz chain ids and endpoint addresses to constants according to docs

### CODE REVIEW

File: constants/chainIds.json [hunk 1]
```diff
@@ -1,19 +1,71 @@
...
+  "sepolia": 10161,
+  "polygon-zkevm": 10158,
+  "scroll": 10170,
+  "tenet": 10173,
+  "canto": 10159,
+  "kava": 10172,
+  "orderly": 10174,
+  "blockgen": 10177,
+  "meritcircle": 10178,
+  "mantle": 10181,
```
Comment (calebwick) [inline constants/chainIds.json:68]: I think you should change this to mantle-testnet

**Comment by Nihar0808** [comment]
Awesome project guys

**Comment by 0xb10ckdev** [comment]
there are duplicated keys

File: constants/chainIds.json [hunk 2]
```diff
@@ -1,19 +1,71 @@
...
-  "fantom-testnet":   10112,
-  "meter-testnet":    10156,
-  "zksync-testnet":   10165
-}
\ No newline at end of file
+  
+  "goerli": 10121,
+  "bsc-testnet": 10102,
+  "fuji-avalanche": 10106,
+  "aptos": 10108,
```
Comment (0xb10ckdev) [inline constants/chainIds.json:39]: some of the keys are duplicated.
it seems it should have the suffix something like `-testnet` for testnets.
e.g. `"aptos-testnet": 10108`

**Comment by TRileySchwarz** [comment]
Currently in the works of importing these ids/chains directly from LZ packages to avoid having these kind of issues in the future.

**Pull request closed without merging**